helm/alfresco-content-services/Chart.yaml

```diff
@@ -85,4 +85,18 @@ dependencies:
     repository: https://helm.elastic.co
     version: 7.17.3
     condition: elasticsearch.enabled
+  - name: elasticsearch
+    alias: elasticsearchAudit
```

Comment:
I'll be extra painful here but this alias is the only one using camelCase. Please switch it to `elasticsearch-audit`